fiware-region-sanity-tests/Dockerfile

```diff
@@ -1,6 +1,7 @@
+# This dockerfile is used for development of phonehome server (commons/http_phonehome_server.py)
```

Comment:
go to line 3 and 4: https://github.com/telefonicaid/fiware-health/pull/291/files/8b6a544f5866a217e6bcbc1d5b798d450700e632#diff-57cfc6ba8a9273646643872aeeda133aR3